### PR TITLE
Nivo Storybook component: LineBarToggle

### PR DESCRIPTION
**Pull request by csong202**
For reference: Issue #1009

### CODE REVIEW

File: client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  title: "charts/NivoLineBarToggle",
+  component: NivoLineBarToggle,
+};
+
+const Template = (args) => <NivoLineBarToggle {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  legend_title: "Legend Title",
+  bar: true,
```
Comment (ktw1016) [inline client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js:20]: I can't help but notice some of these props are obsolete, like `bar` and the options in `graph_options`. And rest of the codebase is still using them who knows for what reasons.. It'd be nice if you could make a separate branch to clean up obsolete props ! (This one can just merge as is I think)

**Review by ktw1016 — APPROVED** [review]

File: client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js (hunk 1 above)
Comment (csong202) [reply client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js:20]: A separate branch to clean up obsolete props on this component or the whole infobase?

Comment (ktw1016) [reply client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js:20]: Just this component

Comment (csong202) [reply client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js:20]: sounds good 👍

**Pull request merged**